src/Elewant/Herding/Application/ProcessManagers/NewUserSubscriber.php

```diff
@@ -24,22 +24,22 @@ final class NewUserSubscriber implements EventSubscriberInterface
      * @todo Because we don't have a Shepherd aggregate, just a ShepherdId, we cannot look it up right now.
      * @todo I suggest we create a Shepherd and ShepherdCollection, so we can use the latter here.
      * @todo We can then do: UserHasRegistered -> GiveBirthToShepherd -> ShepherdWasBorn -> FormHerd.
-     *
      * @var HerdRepository
      */
     private $herdRepository;
 
-    /**
-     * @var CommandBus
-     */
+    /** @var CommandBus */
     private $commandBus;
 
     public function __construct(HerdRepository $herdRepository, CommandBus $commandBus)
     {
         $this->herdRepository = $herdRepository;
-        $this->commandBus     = $commandBus;
+        $this->commandBus = $commandBus;
     }
 
+    /**
+     * @return mixed[]
```

Comment:
This is regarding the rule `SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingTraversableReturnTypeHintSpecification`, which states that you should be more specific that `array` (or `Traversable`, etc). I like this rule, because in most cases you _can_ be more specific (`string[]` instead of `array` for example).

In this case, we _cannot_ be more specific, because several array-structures are allowed.

`@return mixed[]` is kind of a workaround, `mixed` is deemed specific enough. The alternative is suppressing the rule, but I want to minimize that.

PS: I'm not sure if phpcs looks at the parent in these cases. In this specific case the parent is `Symfony\Component\EventDispatcher\EventSubscriberInterface`, isn't more specific that `@return array`, so there is no "correct" parent to fall back to.